src/ui/pages/Dashboard/Wallet.tsx

```diff
@@ -0,0 +1,83 @@
+import { useState } from 'react'
+import Button from '../../components/Button'
+
+// @tkmct - I'm assuming we can convert to USD to start with
```

Comment:
Yeah, but tokens on the Testnet don't have USD values😓

Reply:
Damn, that sucks lol